index.bs

```diff
@@ -136,27 +136,32 @@ primary use case of interactivity.
 ## Intended Audience ## {#intended-audience}
 
 The SIMID standard specification is geared toward the digital media
-(video/audio) advertising community. Anyone who is involved can benefit
-from reading the introductory sections. The API and implementation details
-sections are predominantly targeted to engineers developing ad creation
-or serving products with SIMID interactivity.
+advertising community. Anyone who works with products or services related to digital media advertising can benefit
+from reading the Introduction sections of this spec. The Messaging Protocol, API Reference, and later sections are
+predominantly targeted at software engineers developing products
+or services which leverage the technical capabilities provided by SIMID.
 
 
 # Introduction # {#intro}
 
 Throughout this document, the SIMID interactive component is referred to
 as a “SIMID creative” or “creative”.
 
-Compliance with SIMID requires support for all features discussed in this document, unless a feature is noted as optional. 
+Compliance with SIMID requires support for all features and behaviors specificied in this document, unless a given feature or behavior is explicitly designated as optional.
 Standard RFC language will be used. See <a href="https://tools.ietf.org/html/rfc2119">https://tools.ietf.org/html/rfc2119</a>
 for RFC 2119 for enforcement terminology used in this standard.
 
 ## SIMID Interactive Creative Nature ## {#sivic-interactive-creative-nature}
 
 <img width="100%" src="images/simid_diagram_3_assets.png" alt="A diagram showing SIMID where and how sivic video and html assets load.">
```

Comment:
since you're fixing, can you change sivic to SIMID here?

Reply:
I think you noticed after making this comment that I caught and changed that 'sivic' to 'simid' in a separate PR #275 (along with a bunch of other anchor links).
https://github.com/InteractiveAdvertisingBureau/SIMID/pull/275/files